resources/views/components/torrent/comment-listing.blade.php

```diff
@@ -0,0 +1,72 @@
+@props([
+    'comment',
+])
+
+<article class="comment" id="comment-{{ $comment->id }}" x-data>
+    <header class="comment__header">
+        <time
+            class="comment__datetime"
+            datetime="{{ $comment->created_at }}"
+            title="{{ $comment->created_at }}"
+        >
+            {{ $comment->created_at?->diffForHumans() }}
+        </time>
+
+        <span class="comment__topic">
```

Comment:
Thinking a switch here would be better. Something like
```php
<span class="comment__topic">
    On
    @switch(basename(str_replace('\\', '/', $comment->commentable_type)))
        @case('Torrent')
            Torrent
            <a href="{{ route('torrents.show', ['id' => $comment->commentable->id]) }}">
                {{ $comment->commentable->name }}
            </a>
            @break

        @case('TorrentRequest')
            Request
            <a href="{{ route('requests.show', ['torrentRequest' => $comment->commentable->id]) }}">
                {{ $comment->commentable->name }}
            </a>
            @break

        @default
            Unknown
    @endswitch
</span>
```

Use language files of course. `{{ __('torrent.torrent') }}`

Reply:
Or actually probably the following since the fully qualified path to the related class name is saved:

```blade
@switch($comment->commentable_type)
    @case(\App\Models\Torrent::class)
```

Reply:
Done with:
> Or actually probably the following since the fully qualified path to the related class name is saved:
> 
> ```
> @switch($comment->commentable_type)
>     @case(\App\Models\Torrent::class)
> ```